src/modules/todo.js

```diff
@@ -0,0 +1,60 @@
+import trash from '../trash.svg';
+
+export default class Todo {
+  constructor() {
+    this.todoList1 = JSON.parse(localStorage.getItem('todoList')) || [];
```

Comment:
JSON.parse() and localStorage.getItem("...") are being used many times? what if there is a way to reduce this redundancy? like creating a module for JSON methods and a module for localStorage methods.
I am interested in your thoughts about that.